chicken-sloths/bangazon-cli-sprint2
# option trees for addProductToOrder and createCustomer

# Description
Creates option trees/menus for main menu options:
- 1 (customer)
- 5 (addProduct)
- 7 (deleteProduct)

## Related Ticket(s)
#5, #1, #7 

## Expected Behavior
The CLI should basically repeat what you say.

## Steps to Test Solution
- `npm install`
- `npm start`
- `1` to check the customer route
    - Fill in all the prompts and it should spit back out a Customer object.
- `5` to check the product route
    - There should be a placeholder for the list of all products.
    - Enter a number and it should spit back the number labeled as the ID of the product to add to the order/cart.
- `7` to check the deleteProduct route
    - There should be a placeholder for the list of the active customer's products.
    - Enter a number and it should spit back the number labeled as the ID of the product to delete.

## Testing


## Documentation
